### PR TITLE
Add option to enable statediffing for the fixturenet geth.

### PR DESCRIPTION
**Pull request by telackey**
Add a new environment variable, RUN_STATDIFF, to enable statediffing on a geth node in the fixturenet.  This enabled by default on geth-2.

### CODE REVIEW

File: compose/docker-compose-db.yml [hunk 1]
```diff
@@ -1,20 +1,32 @@
...
+      DATABASE_NAME: "cerc_testing"
+      DATABASE_PASSWORD: "password"
+      DATABASE_HOSTNAME: "ipld-eth-db"
+      DATABASE_PORT: 5432
+
   ipld-eth-db:
+    image: timescale/timescaledb:latest-pg14
     restart: always
-    image: cerc/ipld-eth-db:local
+    command: ["postgres", "-c", "log_statement=all"]
```
Comment (dboreham) [inline compose/docker-compose-db.yml:20]: This probably isn't exactly what we need (log all statements). I'd just leave out the `command` directive until we have time to configure the db properly.

File: compose/docker-compose-fixturenet-eth.yml [hunk 2]
```diff
@@ -21,10 +21,14 @@ services:
 
   fixturenet-eth-geth-2:
     hostname: fixturenet-eth-geth-2
+    environment:
+      RUN_STATEDIFF: "true"
```
Comment (dboreham) [inline compose/docker-compose-fixturenet-eth.yml:25]: Let's prefix all environment variables we invent with some consistent prefix so we know they are ours. I think we are already using `CERC_` elsewhere.

**Review by dboreham — CHANGES_REQUESTED** [review]

File: compose/docker-compose-db.yml (hunk 1 above)
Comment (telackey) [reply compose/docker-compose-db.yml:20]: I can do that, this was just what the original file had.

File: compose/docker-compose-fixturenet-eth.yml (hunk 2 above)
Comment (telackey) [reply compose/docker-compose-fixturenet-eth.yml:25]: Hmm, that might deserve a separate PR, because it would apply to everything in fixturenet-eth.env.

**Pull request merged**